scripts/values.yaml

```diff
@@ -17,3 +20,8 @@ opentelemetry-operator:
   enabled: true
   createDefaultInstrumentation: true
   instrumentationNamespaces: "sumologic"
+kube-prometheus-stack:
+  prometheusOperator:
+    enabled: true
+  prometheus:
+    enabled: false
```

Comment:
Why `false` here? If we're doing this, we should mirror Prometheus itself as well, right?